Module not found: Can't resolve `fs` (Can't use the API on a NextJS Project)

Hi, I'm coming to you because I'd like to integrate the API into a NextJS project, but I'm getting an error when I try to use it and I don't know where it's coming from...

```
Failed to compile

./node_modules/threads-api/build/threads-api.js:1:175
Module not found: Can't resolve 'fs'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/threads-api/build/index.js
./app/api/getUser.tsx
./app/page.tsx
```

This error occurred during the build process and can only be dismissed by fixing the error.

Here is my code : 

```ts
import { NextResponse } from 'next/server';
import { ThreadsAPI } from 'threads-api';


export const getUser = async (): Promise<NextResponse> => {
  const threadsAPI = new ThreadsAPI();
  const username: string = 'zuck';
  const userID: string | null = await threadsAPI.getUserIDfromUsername('unyxes');
  
  if (!userID) {
    console.log(NextResponse.json({})); // Log the empty JSON response
  }
  
  const user = await threadsAPI.getUserProfile(username, userID as string);
  
    console.log(NextResponse.json({ user }));
};
```

--- a/threads-api/src/threads-api.ts
+++ b/threads-api/src/threads-api.ts
@@ -1,5 +1,4 @@
 import axios, { AxiosRequestConfig } from 'axios';
-import * as fs from 'fs';
 import mimeTypes from 'mime-types';
 import { v4 as uuidV4 } from 'uuid';
 import {
@@ -613,6 +612,7 @@ export class ThreadsAPI {
 
     const isFilePath = !imagePath.startsWith('http');
     if (isFilePath) {
+      const fs = await import('fs');
       content = await fs.promises.readFile(imagePath);
       const mimeTypeResult = mimeTypes.lookup(imagePath);
       mime_type = mimeTypeResult ? mimeTypeResult : 'application/octet-stream';

